components/Player/Player.jsx

```diff
@@ -88,15 +88,15 @@ const mapDispatchToProps = (dispatch) => ({
 class RequestLayer extends React.Component {
   componentDidMount() {
     this.props.getPlayer(this.props.playerId);
```

Comment:
at some points, should probably refactor this and the very similar code below to deduplicate.